src/main/resources/config/liquibase/changelog/20230519181800_changelog.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.18.xsd">
+    <changeSet author="kotzian" id="20230519181800">
+        <addColumn tableName="exercise">
+            <column name="iris_activated" type="boolean">
+                <constraints nullable="true"/>
```

Comment:
I think making this nullabe would make sense. This is a simple true/false option where the default is false.